jetty-servlet/src/main/java/org/eclipse/jetty/servlet/ServletContextHandler.java

```diff
@@ -652,16 +622,6 @@ public void setSecurityHandler(SecurityHandler securityHandler)
         relinkHandlers();
     }
 
-    /**
-     * @param gzipHandler The {@link GzipHandler} to set on this context.
-     */
-    public void setGzipHandler(GzipHandler gzipHandler)
```

Comment:
I think we should leave the get and set methods, but deprecate them.  The implementation can be replaced by an insert handler call